Project.toml

```diff
@@ -40,6 +51,8 @@ DocStringExtensions = "0.8, 0.9"
 ForwardDiff = "0.10"
 KernelAbstractions = "0.8"
```

Comment:
```suggestion
KernelAbstractions = "0.9"
```

Reply:
No KernelAbstractions 0.9 is a breaking release and will require more work that that

Reply:
But https://github.com/JuliaGPU/KernelAbstractions.jl/pull/337 is in v0.9?

Reply:
No that was merged onto `release-0.8`

Reply:
🤔 https://github.com/JuliaGPU/KernelAbstractions.jl/releases/tag/v0.9.0 okay that's just wrong then.

Reply:
Silly tagbot.